pcg/src/main/kotlin/nl/perfana/config/generator/PerfanaConfigGeneratorController.kt

```diff
@@ -71,27 +78,41 @@ class PerfanaConfigGeneratorController(val storage: FileStorage) {
             }
         } finally {
             val output = extractOutput(process)
-            print(output)
+            println(output)
             process.destroy()
         }
     }
 
     private fun extractOutput(process: Process): String {
         val reader = BufferedReader(InputStreamReader(process.inputStream))
-        val output = reader.lines().collect(Collectors.joining(System.getProperty("line.separator")))
+        val output = reader.lines().collect(Collectors.joining(System.lineSeparator()))
         return output
     }
 
-    @GetMapping("/download/{projectId:.+}")
-    fun download(@PathVariable projectId: String, request: HttpServletRequest): ResponseEntity<Resource> {
+
+    @PostMapping("/generate")
```

Comment:
the frontend code also needs to call this now (plan is to also create a endpoint without a zip file to be used by frontend)

Reply:
We can keep the /upload and /download methods too, for now. I will put them back.